backend/python/tools/insert_test_data.py

```diff
@@ -0,0 +1,341 @@
+                level=4,
+                translated_languages=["ENGLISH_UK"],
+            ),
+            Story(
+                title="Nineteen Eighty-Four",
+                description="Nineteen Eighty-Four, often referred to as 1984, is a dystopian social science fiction novel by the English novelist George Orwell.",
+                youtube_link="https://www.youtube.com/watch?v=h9JIKngJnCU",
+                level=2,
+                translated_languages=[],
+            ),
+        ]
+    )
+    db.session().commit()
+    generic_content = [
+        "Every two weeks I went to a meeting with them, and in my room here I covered pages with writing. I bought every known Hebrew dictionary. But the old gentlemen were always ahead of me. It wasn't long before they were ahead of our rabbi; he brought a colleague in.",
+        "Mr. Hamilton, you should have sat through some of those nights of argument and discussion. The questions, the inspection, oh, the lovely thinking-the beautiful thinking.",
+        "After two years we felt that we could approach your sixteen verses of the fourth chapter of Genesis. ",
+        "My old gentlemen felt that these words were very important too-'Thou shalt' and 'Do thou.' And this was the gold from our mining: 'Thou mayest.' 'Thou mayest rule over sin.' ",
+        'The old gentlemen smiled and nodded and felt the years were well spent. It brought them out of their Chinese shells too, and right now they are studying Greek."',
+        "Samuel said, \"It's a fantastic story. And I've tried to follow and maybe I've missed somewhere. Why is this word so important?\"",
+        "Lee's hand shook as he filled the delicate cups. He drank his down in one gulp. \"Don't you see?\" he cried. ",
+        '"The American Standard translation orders men to triumph over sin, and you can call sin ignorance. ',
+        "The King James translation makes a promise in 'Thou shalt,' meaning that men will surely triumph over sin. ",
+        "But the Hebrew word, the word timshel-'Thou mayest'-that gives a choice. It might be the most important word in the world. That says the way is open. That throws it right back on a man. For if 'Thou mayest'-it is also true that 'Thou mayest not.' Don't you see?\"",
+    ]
+
+    # story contents
+    db.engine.execute("ALTER TABLE story_contents AUTO_INCREMENT = 1;")
+    for story_id in range(1, 8):
+        for i, content in enumerate(generic_content):
+            db.session.add(
+                StoryContent(
+                    story_id=story_id,
+                    line_index=i,
+                    content=content,
+                ),
+            )
+
+    # story translations
+    db.engine.execute("ALTER TABLE story_translations AUTO_INCREMENT = 1;")
+    db.session.bulk_save_objects(
+        [
+            StoryTranslation(
+                story_id=1, language="GERMAN", stage="TRANSLATE", translator_id=5
```

Comment:
@gaoxk was there a specific reason why you used translator_id=5 rather than 6 for this one?